### PR TITLE
Update to 2.0

### PR DESCRIPTION
**Pull request by kroepke**
Graylog 2.0 support
Elasticsearch 2.1 support
Java 7 is no longer supported

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: java
 jdk:
-  - oraclejdk7
```
Comment (joschi) [inline .travis.yml:4]: This should also be updated in the `deploy` section of this file.

File: pom.xml [hunk 2]
```diff
@@ -58,7 +58,7 @@
         <maven.compiler.source>1.7</maven.compiler.source>
         <maven.compiler.target>1.7</maven.compiler.target>
```
Comment (joschi) [inline pom.xml]: No Java 7 &rarr; `1.8`.

File: pom.xml [hunk 3]
```diff
@@ -58,7 +58,7 @@
...
         <tagNameFormat>@{project.version}</tagNameFormat>
-        <graylog.version>1.3.0-20151127.153927-347</graylog.version>
+        <graylog.version>2.0.0-SNAPSHOT</graylog.version>
         <jackson.version>2.5.3</jackson.version>
```
Comment (joschi) [inline pom.xml]: Graylog 2.x is using Jackson 2.6.x (2.6.4 at the time of writing).

File: src/main/java/org/graylog/plugins/usagestatistics/collectors/ElasticsearchCollector.java [hunk 4]
```diff
@@ -69,25 +69,17 @@ public ElasticsearchCollector(Client client, ClusterStatsService clusterStatsSer
...
-                    info.getOs().cpu().coresPerSocket(),
-                    info.getOs().cpu().cacheSize().bytes()
-            );
-            final HostInfo.Memory memory = info.getOs().mem() == null ? null : HostInfo.Memory.create(info.getOs().mem().total().bytes());
-            final HostInfo.Memory swap = info.getOs().swap() == null ? null : HostInfo.Memory.create(info.getOs().swap().total().bytes());
+            final MacAddress macAddress = MacAddress.EMPTY;
+            final HostInfo.Cpu cpu = null;
+            final HostInfo.Memory memory = null;
+            final HostInfo.Memory swap =  null;
             final HostInfo hostInfo = HostInfo.create(macAddress, cpu, memory, swap);
```
Comment (joschi) [inline src/main/java/org/graylog/plugins/usagestatistics/collectors/ElasticsearchCollector.java:79]: The complete `HostInfo` could go if every member is empty or null anyway.

Comment (kroepke) [inline src/main/java/org/graylog/plugins/usagestatistics/collectors/ElasticsearchCollector.java:79]: but currently it's not nullable, i think. I didn't want to touch the backend service right now in, too

Comment (joschi) [inline src/main/java/org/graylog/plugins/usagestatistics/collectors/ElasticsearchCollector.java:79]: :white_check_mark: Fair enough. Maybe add a `@Deprecated` or at least a `TODO` comment to remove this in a future version.

Comment (kroepke) [inline src/main/java/org/graylog/plugins/usagestatistics/collectors/ElasticsearchCollector.java:79]: Makes sense!

**Comment by joschi** [comment]
LGTM. :+1:

**Pull request merged**